src/style.css

```diff
@@ -0,0 +1,128 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-secondary-color);
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
+  --card-background-color: #fff;
+  --card-border-radius: 5px;
+  --card-border: 1px solid #f3f3f3;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: stretch;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: var(--card-border);
+  border-radius: var(--card-border-radius);
+  background-color: var(--card-background-color);
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--text-accent-color);
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  
+  height: 40px;
+  background-color: var(--main-color);
+  border: 1px solid var(--main-color);
+  border-radius: var(--card-border-radius);
+  
+  font-size: 14px;
+  line-height: 16px;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  color: #fff;
+  font-weight: 700;
+}
+
+.card__button:hover {
+  background-color: var(--card-background-color);
+  color: var(--main-color);
+}
+
+.card__pricing {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__pricing-word {
+  font-size: 12px;
+  font-weight: 400;
```

Comment:
it's a default `font-weight`

Reply:
one more little request) start please auto tests for my previous layout_snake
thank you!